src/main/java/net/javadiscord/javabot/systems/help/HelpListener.java

```diff
@@ -160,34 +192,35 @@ private boolean doesMatchDetector(String content, String[] detector) {
 	}
 
 	private Button createCloseSuggestionButton(ThreadChannel post) {
-		return Button.primary(ComponentIdBuilder.build(HelpManager.HELP_CLOSE_IDENTIFIER, post.getIdLong()), "Close Post");
-	}
-
-	@Override
-	public void onChannelCreate(@NotNull ChannelCreateEvent event) {
-		if (isInvalidForumPost(event.getChannel())) {
-			return;
-		}
-		ThreadChannel post = event.getChannel().asThreadChannel();
-		if (isInvalidHelpForumChannel(post.getParentChannel().asForumChannel())) {
-			return;
-		}
-		// add thread id to a temporary cache to avoid potential missing author message
-		// more info on why this has to be done: https://canary.discord.com/channels/125227483518861312/1053705384466059354/1053705384466059354 (JDA Discord)
-		newThreadChannels.add(event.getChannel().getIdLong());
+		return Button.primary(ComponentIdBuilder.build(
+				HelpManager.HELP_CLOSE_IDENTIFIER,
+				post.getIdLong()
+		), "Close Post");
 	}
 
 	@Override
 	public void handleButton(@NotNull ButtonInteractionEvent event, @NotNull Button button) {
 		String[] id = ComponentIdBuilder.split(event.getComponentId());
-		if (isInvalidForumPost(event.getChannel()) || isInvalidHelpForumChannel(event.getChannel().asThreadChannel().getParentChannel().asForumChannel())) {
-			Responses.error(event, "This button may only be used inside help forum threads.").queue();
+		if (isInvalidForumPost(event.getChannel()) || isInvalidHelpForumChannel(event.getChannel()
```

Comment:
I don't think it makes much sense to split a subexpression into multiple lines when there's a different expression in the same line - maybe add another line break directly after the `||`